pkg/settings/migration.go

```diff
@@ -94,12 +93,13 @@ func (r *Migration) Load() (err error) {
 		return liberr.Wrap(err)
 	}
 	if virtV2vImage, ok := os.LookupEnv(VirtV2vImage); ok {
-		if cold, warm, found := strings.Cut(virtV2vImage, "|"); found {
-			r.VirtV2vImageCold = cold
-			r.VirtV2vImageWarm = warm
+		if img, _, found := strings.Cut(virtV2vImage, "|"); found {
+			// previously, we used different images for cold and warm migration and these
+			// could be specified separately as "COLD_IMAGE|WARM_IMAGE". Accept this syntax,
+			// but just ignore the second image
```

Comment:
I see that you also removed the warm from `VIRT_V2V_IMAGE`. Think this is no longer needed.